src/js/containers/publish-page.js

```diff
@@ -0,0 +1,76 @@
+/*eslint-disable no-unused-vars*/
+import React, { Component } from 'react';
+import { Paper, FontIcon, RaisedButton } from 'material-ui';
+import { Page } from '../components/page';
+
+import PointList from '../components/point-list';
+/*eslint-enable no-unused-vars*/
+
+import { selectMarker } from '../actions/map-actions';
+import { setDrawer } from '../reducers/drawer';
+import { connect } from 'react-redux';
+
+import history from '../history';
+
+class PublishPage extends Component {
+  componentDidMount() {
+    this.props.dispatch( setDrawer( 'Publish Services' ) );
```

Comment:
Publish Services -> Publish (since we're really publishing alerts, services, and comments)